### ISSUE TITLE
[feature] Automatic downloads for VQA2, COCO, TextVQA, VizWiz

### ISSUE THREAD

**Issue by apsdehal** [facebookresearch/mmf#165]
## 🚀 Feature

VisualGenome PR #140 introduces automatic downloads for Visual Genome features and IMDb. We need to add the same support for the other datasets mentioned in the title.

**Comment by Patil2099** [comment]
@apsdehal Can I work on this issue?

**Comment by apsdehal** [comment]
@Patil2099 Sure. Go ahead.

**Comment by Patil2099** [comment]
@apsdehal Can you help me a little bit?
Thanks!

**Comment by apsdehal** [comment]
@Patil2099 Do you have any specific questions?

**Comment by Patil2099** [comment]
@apsdehal Do I have to make .yml file or I have to do more?
Thanks!

**Comment by apsdehal** [comment]
@Patil2099 Currently, for training on the datasets mentioned in the title of this issue, one has to first download imdbs, features and vocabularies for them which is overall tedious and cumbersome. Ideally, running the training command should automatically download everything needed for training. In VisualGenome, for the first time we added this capability. We want to replicate the same functionality in these other datasets too. So, it requires code changes (instead of any kind of .yml changes) in builder.py files for these datasets similar to what we did in VisualGenome PR.
